src/components/src/ComplianceSupport/compliancesupport.main.tsx

```diff
@@ -134,26 +135,44 @@ class ComplianceSupportModal extends Component<ComplianceSupportModalProps, Comp
     return (
       <div>
         <Modal open={open} onClose={this.handleCloseModal}>
-          <div className="modal-lg compliance-support-modal">
+          <div className="compliance-support-modal">
             <div className="modal-content">
               <div className="modal-header">
                 <h2 className="modal-title">
-                  {intl.get('compliance')}
+                  {intl.get('your-compliance-notice')}
                 </h2>
               </div>
               <div className="modal-body">
                 <div className="compliance-checkbox-wrap">
                   <label htmlFor="compliance_agreement">
-                    <input id="compliance_agreement" type="checkbox" onChange={this.handleCheck} defaultChecked={checked} />
-                    <span className="helping-el" />
-                    <span className="label-text">
-                      {intl.get('compliance-label')}
-                    </span>
+                    <div className="first-label">
```

Comment:
I'm just wondering if there's an easier way to do this using placeholder text. So that a consumer of this can provide a large blob of text with mustache syntax (or something else) and a replacement is done to inject the links.
Example:
"As a website owner you may need to comply with national or international privacy {{privacy_link}} laws."